builtin/game/chatcommands.lua

```diff
@@ -530,11 +530,16 @@ core.register_chatcommand("giveme", {
 })
 
 core.register_chatcommand("spawnentity", {
-	params = "<EntityName>",
-	description = "Spawn entity at your position",
+	params = "<EntityName> [<X>,<Y>,<Z>]",
```

Comment:
Why is it inconsistent? chatcommands.lua has the number of parameters per command varying.